### PR TITLE
Organizers offline first caching and some minor refactors

### PR DESCRIPTION
**Pull request by chege4179**
I cached the organizers from the API, refactored some code and renamed some of the existing files for consistency

### CODE REVIEW

File: data/src/main/java/com/android254/data/work/SyncDataWorker.kt [hunk 1]
```diff
@@ -58,87 +53,35 @@ class SyncDataWorker @AssistedInject constructor(
...
-
-                is ResourceResult.Error -> {
-                    Timber.d("Sync speakers failed ${response.message}")
-                }
-
-                else -> {}
-            }
-        }
-    }
+//    private suspend fun startForegroundService(notificationInfo:String) {
```
Comment (wangerekaharun) [inline data/src/main/java/com/android254/data/work/SyncDataWorker.kt:74]: Maybe cleanup if this is no longer needed?

**Review by wangerekaharun — APPROVED** [review]
Looks good, left some comments

Also do the same for feed

File: data/src/main/java/com/android254/data/work/SyncDataWorker.kt (hunk 1 above)
Comment (chege4179) [reply data/src/main/java/com/android254/data/work/SyncDataWorker.kt:74]: Let me remove it 
I forgot

**Comment by chege4179** [comment]
@wangerekaharun What do you mean feed ....The whole feed feature

**Comment by wangerekaharun** [comment]
> @wangerekaharun What do you mean feed ....The whole feed feature

we need to add feeds to the offline first approach

**Comment by chege4179** [comment]
The Feed Api is not returning any json

**Comment by wangerekaharun** [comment]
I'll ping you when it's fixed

**Comment by chege4179** [comment]
Ok 
should I just proceed with the code changes without testing or should i wait until its fixed for me to make the changes

**Comment by wangerekaharun** [comment]
> Ok should I just proceed with the code changes without testing or should i wait until its fixed for me to make the changes

we will do it separately, for now happy to merge this

**Comment by chege4179** [comment]
Let me push that small change first

**Comment by wangerekaharun** [comment]
@chege4179 some minor conflict

**Comment by chege4179** [comment]
@wangerekaharun 
Resolved conflicts

**Review by chepsi — APPROVED** [review]

**Pull request merged**